contents/data_vis_makie_recipe.md

```diff
@@ -0,0 +1,96 @@
+## A Makie recipe for a DataFrame {#sec:recipe_df}
+
+Unlike other libraries that already support a wide set of input formats via recipes, i.e. `Plots.jl`, in `Makie.jl` most of the time we need to pass the raw data to our functions.
+However, if we want to define our own `recipe` in Makie we could also do it.
+As an example we will code a small **full recipe** for a `DataFrame`. Please refer to the [documentation](https://makie.juliaplots.org/stable/documentation/recipes/) for more details.
```

Comment:
Can you elaborate a bit more on what a recipe is? To be honest, I'm also not sure whether I fully understand it. As far as I understand, recipes are meant for lightweight interface packages so that you can define plotting in your package without having to load the full plotting library? Also, it allows defining a basic framework?

Reply:
Yeah I agree with @rikhuijzer, recipes need at least a simple definition.

Reply:
I also agree on this 😄 ... I will come back shortly.

Reply:
An extra paragraph is up.